{ 'stand_up_back_300': { 'head': [],
                         'left_arm': [ { 'duration': 1.0,
                                         'name': u'stand_up_back_950',
                                         'positions': [ -0.4775,
                                                        0.3868,
                                                        0.0007,
                                                        -0.1045,
                                                        -1.5972,
                                                        -1.0048,
                                                        0.0023],
                                         'starttime': 0.0},
                                       { 'duration': 1.011,
                                         'name': u'stand_up_back_1000',
                                         'positions': [ -0.3855,
                                                        0.0039,
                                                        -0.001,
                                                        -0.0026,
                                                        -0.9922,
                                                        -0.0005,
                                                        0.0038],
                                         'starttime': 1.0}],
                         'left_leg': [ { 'duration': 1.989,
                                         'name': u'stand_up_back_1000',
                                         'positions': [ 0.0017,
                                                        0.0006,
                                                        0.6242,
                                                        2.0027,
                                                        1.7986,
                                                        -0.0001],
                                         'starttime': 0.0},
                                       { 'duration': 2.008,
                                         'name': u'stand_up_back_1100',
                                         'positions': [ -0.0019,
                                                        -0.0006,
                                                        0.0176,
                                                        2.5058,
                                                        1.7976,
                                                        -0.0001],
                                         'starttime': 1.989}],
                         'right_arm': [ { 'duration': 1.0,
                                          'name': u'stand_up_back_950',
                                          'positions': [ 0.4775,
                                                         -0.3868,
                                                         -0.0007,
                                                         0.1045,
                                                         1.5972,
                                                         1.0048,
                                                         -0.0023],
                                          'starttime': 0.0},
                                        { 'duration': 1.018,
                                          'name': u'stand_up_back_1000',
                                          'positions': [ 0.3855,
                                                         -0.0039,
                                                         0.001,
                                                         0.0026,
                                                         0.9922,
                                                         0.0005,
                                                         -0.0038],
                                          'starttime': 1.0}],
                         'right_leg': [ { 'duration': 1.988,
                                          'name': u'stand_up_back_1000',
                                          'positions': [ -0.0017,
                                                         -0.0006,
                                                         -0.6242,
                                                         -2.0027,
                                                         -1.7986,
                                                         0.0001],
                                          'starttime': 0.0},
                                        { 'duration': 2.008,
                                          'name': u'stand_up_back_1100',
                                          'positions': [ 0.0019,
                                                         0.0006,
                                                         -0.0176,
                                                         -2.5058,
                                                         -1.7976,
                                                         0.0001],
                                          'starttime': 1.988}],
                         'torso': [ { 'duration': 2.0,
                                      'name': u'stand_up_back_1100',
                                      'positions': [-0.0002, 0.6006],
                                      'starttime': 0.0}]}}
